.travis.yml

```diff
@@ -1,12 +1,21 @@
 language: rust
+
 rust:
   - stable
   - beta
   - nightly
+
 os:
   - linux
 
 matrix:
   fast_finish: true
+
 notifications:
   email: false
+
+deploy:
```

Comment:
Can we just make this:

```
script:
  scripts/deploy.sh
```

and maybe just rename `scripts/deploy.sh` to `test/run.sh`?
  
```